CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 
 ### Development
 
+* Update manifest permissions to allow Fetch requests from Firefox addon,
+  closes [issue #197](https://github.com/refined-bitbucket/refined-bitbucket/issues/197),
+  [pull request #198](https://github.com/refined-bitbucket/refined-bitbucket/pull/198).
```

Comment:
This should go under the "Bug Fixes" header. Just add it between "Features" and "Development"